Plugins/Public/event/Main.cpp

```diff
@@ -556,16 +583,16 @@ void LoadSettings()
 	ConPrint(L"EVENT DEBUG: Loaded %u event player data\n", iLoaded3);
 
 	LoadIDs();
-
+	HkLoadStringDLLs();
 }
 
 ///////////////////////////////////////////////////////////////////////////////////////////////////////////////
 //Functions
 ///////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
-FILE *Logfile = fopen("./flhook_logs/event_log.log", "at");
```

Comment:
In the future, please try to avoid such formatting changes bleeding into the PR when the line is otherwise untouched.